### PR TITLE
New blog post on how to contribute to OTel

### PR DESCRIPTION
**Pull request by avillela**
Blog post on how to contribute to OTel as discussed with @svrnm 

Related: #3254

### CODE REVIEW

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:97]:
```suggestion
### Contribute to the OpenTelemetry Demo
```

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:124]: again, add your [own contributions](https://github.com/open-telemetry/opentelemetry-demo/pull/432), so people have something tangible, especially this one ticket shows that contributing is a little bit of work, but you get feedback and over some time it gets merged

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:125]:
```suggestion
### Join a Special Interest Group
```

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:145]:
```suggestion
### Final thoughts
```

File: content/en/blog/2023/contributing-to-otel/turks-sunset.jpg
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/turks-sunset.jpg:1]: Can you scale this image down, with ~840KB it is fairly big and even on a large screen it is scaled down significantly on the page itself

**Review by svrnm — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:44]:
```suggestion
###Contribute to the OTel Docs
```

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:182]: It's maybe worth to emphasis that there are also contributions like "giving talks about OpenTelemetry", "telling your friends and family (and co-workers) how awesome OpenTelemetry is", so also things outside the repositories and official infrastructure (slack, zooms, cncf thingies) is highly helpful

**Review by svrnm — CHANGES_REQUESTED** [review]
looks good to me overall, 2 more comments

File: content/en/blog/2023/contributing-to-otel/index.md
Comment (svrnm) [inline content/en/blog/2023/contributing-to-otel/index.md:4]: Let's maybe have this blog post on Monday, since we had the PHP one just today

```suggestion
date: 2023-09-18
```

**Review by svrnm — APPROVED** [review]
looks good to me now!

@open-telemetry/docs-approvers any more comments? PTAL before merge!:)

**Review by cartermp — APPROVED** [review]

**Comment by svrnm** [comment]
@cartermp publication date is set to `date: 2023-09-18` so we have to make sure that something gets merged on Monday to make the blog show up

**Pull request merged**
